prototype/mc_shortest_path/arduino_mc_shortest_path/mc_shortest_path.py

```diff
@@ -0,0 +1,35 @@
+import struct
+import time
+import serial
+import numpy as np
+
+def main():
+    np.set_printoptions(suppress=True) # Prevents from printing in scientific notation
+
+    port = serial.Serial('/dev/tty.usbmodem2101', 9600)
+    time.sleep(2)
+
+    size = 9
+    port.write(struct.pack('<B', size)) # Make sure that size in the arduino program is of 1 byte
+    
+    boardValues = [1, 2, 3, 1, 2, 4, 5, 2, 1]
+    for value in boardValues:
+        port.write(struct.pack('<b', value))
+
+    nPlayers = 0
+    port.write(struct.pack('<B', nPlayers))
+
+#Retrieve eigenvalues and eigenvectors
+    print('Trying to read from port')
+
+    xPos = port.read()
```

Comment:
You can directly read bytes using struct.unpack, please check the other prototypes for examples